prov/gni/src/gnix_vc.c

```diff
@@ -2079,9 +2078,10 @@ static int __gnix_ep_rdm_get_vc(struct gnix_fid_ep *ep, fi_addr_t dest_addr,
 				   fi_strerror(-ret));
 			goto err;
 		}
-	}
+	} else
```

Comment:
Can you use braces around this single statement?  The new linux style checker should allow it.